lte/gateway/c/core/oai/tasks/nas/esm/sap/esm_recv.c

```diff
@@ -431,13 +431,14 @@ esm_cause_t esm_recv_pdn_disconnect_request(
       PARENT_STRUCT(emm_context, struct ue_mm_context_s, emm_context);
 
   if (!ue_mm_context_p) {
-    OAILOG_WARNING(
-        LOG_NAS_ESM, "Failed to find ue context from emm context \n");
+    OAILOG_WARNING_UE(
```

Comment:
ue_mm_context does not exist in this case, let's not change this

Reply:
Resolved